src/components/ReplyField/ReplyFieldContainer.tsx

```diff
@@ -15,6 +15,7 @@ export const mapStateToProps = (state: AppState): Props => ({
 
 export const mapDispatchToProps = {
     setCursorPosition: setCursorAction,
+    updateCollaborators: fetchCollaboratorsAction,
```

Comment:
In CRUD, "update" usually means to edit/overwrite. Can we keep this as `fetchCollaborators: fetchCollaboratorsAction` for simplicity?